src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ReadyToRunHelperNode.cs

```diff
@@ -27,7 +28,7 @@ public enum ReadyToRunHelperId
         GenericLookupFromDictionary,
     }
 
-    public partial class ReadyToRunHelperNode : AssemblyStubNode
+    public partial class ReadyToRunHelperNode : AssemblyStubNode, INodeWithRuntimeDeterminedDependencies
```

Comment:
Note to self: might want to just have a separate node for the generic lookup helpers. Those are the only ones that actually have runtime determined dependencies...